Makefile

```diff
@@ -20,6 +20,7 @@ payments: spec=PaymentService-v68
 recurring: spec=RecurringService-v68
 payout: spec=PayoutService-v68
 management: spec=ManagementService-v1
+management: resourceClass=Management
```

Comment:
```suggestion
management: spec=Management
```

Reply:
This variable `resourceClass` is something new, I've added it for the upcoming generated service classes.

Reply:
Yep I merged that branch into this one :)